tgstation.dme

```diff
@@ -5282,10 +5285,21 @@
 #include "hl13\code\modules\modular_computers\computers\item\pda.dm"
 #include "hl13\code\modules\modular_computers\file_system\programs\civilian_data.dm"
 #include "hl13\code\modules\modular_computers\file_system\programs\socio_stability_monitor.dm"
+#include "hl13\code\modules\projectiles\ammunition\ballistic\ar2.dm"
```

Comment:
Optimize all imports after you make the edits!

Reply:
yep